scripts/get_default_keybindings/main.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+const path = require('path');
+const fsPromises = require('fs/promises');
+const { runTests } = require('vscode-test');
+
+async function main() {
+    try {
+        // Make two empty directories to use to launch VS Code with the cleanest possible profile.
+        const emptyDir1 = path.resolve(__dirname, 'empty1');
```

Comment:
In case I missed someplace where it's already been done, consider deleting the contents and/or assert that it's empty. Just in the unlikely case an artifact from another version affects anything!

Reply:
Agreed, I think a temp path would work better here, I might make this change

Reply:
Good catch! Thanks.